pkgs/by-name/ki/kickoff/package.nix

```diff
@@ -38,7 +38,7 @@ rustPlatform.buildRustPackage rec {
   ];
 
   postInstall = ''
-    wrapProgram "$out/bin/kickoff" --prefix LD_LIBRARY_PATH : "${libPath}"
+    wrapProgram "$out/bin/kickoff" --prefix LD_LIBRARY_PATH : "${finalAttrs.libPath}"
```

Comment:
I'd argue this is actually fine given it's used here and not just in `src.version`.

cc @superherointj, given your -1, what do you think?

Reply:
Thanks for asking.

Considering I care about eval times. My impression on fixed point:

> I also heard that the use of finalAttrs can increase eval time, which is inline with other experiences / benchmarks that I made, fixed points computation like the modules and overlays are slow and we should avoid them when possible. - Mic92

Source: https://github.com/NixOS/nixpkgs/issues/320528#issuecomment-2175087036

`rec` is local and faster. So I'd use fixed point only in the cases of necessity.

I have removed my objection. Feel free to proceed as you wish.